README.md

```diff
@@ -111,34 +98,21 @@ mount / render it into the HTML element you created in step 3.
     }    
     ```
 
-5. Run `pub serve` in the root of your Dart project.
+5. Run `pub run build_runner serve` in the root of your Dart project.
 
 &nbsp;
 
 ### Running tests in your project
 
-When running tests on code that uses our [transformer] _(or any code that imports `over_react`)_,
-__you must run your tests using Pub__.
-
-1. Add the `test/pub_serve` transformer to your `pubspec.yaml` _after_ the `over_react` transformer.
-
-    ```yaml
-    transformers:
-      - over_react
-      - test/pub_serve:
-          $include: test/**_test{.*,}.dart
-      - $dart2js
-    ```
+When running tests on code that uses our [builder] _(or any code that imports `over_react`)_,
+__you must run your tests using build_runner__.
 
-2. Use [the `--pub-serve` option](https://github.com/dart-lang/test#testing-with-barback) when running your tests:
+1. Run tests through build_runner, and specify the platform to be a browser platform. Example: 
 
     ```bash
-    $ pub run test --pub-serve=8081 test/your_test_file.dart
+    $ pub run build_runner test -- -p chrome test/your_test_file.dart
```

Comment:
This doesn't work.  Have to run

```
pub run build_runner test -- -P dartdevc
```
or
```
pub run build_runner test -r -- -P dart2js
```

Reply:
Those presets apply to this project only, and may not be present in consumer repos. `pub run build_runner test -- -p chrome test/your_test_file.dart` should be valid for a generic case. 

I'll update with a section for running tests in this repo as well as the more generic command.